src/Select/Select.tsx

```diff
@@ -67,12 +70,19 @@ export const Select: React.FC<SelectProps> = ({
   const triggerComponent = getTriggerComponent(valueComponentProps, valueComponent)
```

Comment:
What's the reason for using both `triggerComponent` and `valueComponent`? Are they not the same thing? In that case we might want to align the variable/prop names.